CHANGELOG.md

```diff
@@ -1,6 +1,22 @@
 Changelog
 =========
 
+2.0.1
+-----
+
+### Added
+
+* Individual rules in the `cache_control` can now again have a `match_response`
+  or `additional_status` configuration to limit the rule to certain responses. 
```

Comment:
should we call this additional_response_status to be clear? on cacheable, its response.additional_status